lms/djangoapps/discussion/templates/discussion/edx_ace/responsenotification/email/body.html

```diff
@@ -2,33 +2,29 @@
 
 {% load i18n %}
 {% load static %}
-
-{% block preview_text %}
-    {% blocktrans trimmed %}
-        {{ comment_username }} replied to your thread.
-    {% endblocktrans %}
-{% endblock %}
-
 {% block content %}
 <table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
     <tr>
         <td>
-            {% blocktrans trimmed %}
-                <p>
-                    Hi {{ thread_username }},
-                </p>
+            <p>
+                {% blocktrans trimmed %}
+                    {{ comment_username }} replied to your <b>{{ thread_title }}</b> post
+                {% endblocktrans %}
+            </p>
 
-                <p>
-                    {{ comment_username }} made the following reply to {{ thread_title }} at {{ comment_created_at }}.
-                </p>
+            <div style="border: 1px solid white;
+                        padding: 15px 15px 15px 20px;
+                        box-shadow: 0 0px 2px rgba(0, 0, 0, 0.55);
+                        margin: 30px 70px 40px 0px;">
+                {{ comment_body }}
+            </div>
```

Comment:
Ah. Inline styling.

Reply:
It's delightful.

The other ace templates all have inline styling, too, so I'm just letting it happen.